### PR TITLE
Remove stutter

### LINKED ISSUE #6: API stutters a lot
Hi,

while using the `xattr` package I noticed that the API stutters a lot, e.g. the `XAttrError` type needs to be referred to from external code as `xattr.XAttrError`. It would be much nicer to rename it to `Error`, so that the referral would look like `xattr.Error` instead. It's the same with the functions, `xattr.Get()` instead of `xattr.Getxattr()` would reduce the stutter a lot.

Are you interested in a PR which fixes this? If not, please feel free to close this issue (and I'm going to live with the stutter ;).

### PR DESCRIPTION
**Pull request by fd0**
Closes #6

### CODE REVIEW

File: xattr_darwin.go [hunk 1]
```diff
@@ -42,16 +42,16 @@ func Listxattr(path string) ([]string, error) {
 	return []string{}, nil
 }
 
-// Setxattr associates name and data together as an attribute of path.
-func Setxattr(path, name string, data []byte) error {
+// Set associates name and data together as an attribute of path.
+func Set(path, name string, data []byte) error {
 	if err := setxattr(path, name, &data[0], len(data), 0, 0); err != nil {
 		return &XAttrError{"setxattr", path, name, err}
```
Comment (kuba--) [inline xattr_darwin.go]: leftover `XAttrError`

File: xattr_darwin.go [hunk 2]
```diff
@@ -42,16 +42,16 @@ func Listxattr(path string) ([]string, error) {
...
 	}
 	return nil
 }
 
-// Removexattr removes the attribute associated with the given path.
-func Removexattr(path, name string) error {
+// Remove removes the attribute associated with the given path.
+func Remove(path, name string) error {
 	if err := removexattr(path, name, 0); err != nil {
 		return &XAttrError{"removexattr", path, name, err}
```
Comment (kuba--) [inline xattr_darwin.go]: leftover `XAttrError`

File: xattr_freebsd.go [hunk 3]
```diff
@@ -49,8 +49,8 @@ func Listxattr(path string) ([]string, error) {
 	return []string{}, nil
 }
 
-// Setxattr associates name and data together as an attribute of path.
-func Setxattr(path, name string, data []byte) error {
+// Set associates name and data together as an attribute of path.
+func Set(path, name string, data []byte) error {
 	written, err := extattr_set_file(path, EXTATTR_NAMESPACE_USER, name, &data[0], len(data))
 	if err != nil {
 		return &XAttrError{"extattr_set_file", path, name, err}
```
Comment (kuba--) [inline xattr_freebsd.go]: `XAttrError`

File: xattr_freebsd.go [hunk 4]
```diff
@@ -61,8 +61,8 @@ func Setxattr(path, name string, data []byte) error {
 	return nil
 }
 
-// Removexattr removes the attribute associated with the given path.
-func Removexattr(path, name string) error {
+// Remove removes the attribute associated with the given path.
+func Remove(path, name string) error {
 	if err := extattr_delete_file(path, EXTATTR_NAMESPACE_USER, name); err != nil {
 		return &XAttrError{"extattr_delete_file", path, name, err}
```
Comment (kuba--) [inline xattr_freebsd.go]: `XAttrError`

File: xattr_linux.go [hunk 5]
```diff
@@ -4,58 +4,58 @@ package xattr
...
 }
 
-// Removexattr removes the attribute associated
+// Remove removes the attribute associated
 // with the given path.
-func Removexattr(path, name string) error {
+func Remove(path, name string) error {
 	if err := syscall.Removexattr(path, name); err != nil {
-		return &XAttrError{"removexattr", path, name, err}
+		return &Error{"removexattr", path, name, err}
```
Comment (kuba--) [inline xattr_linux.go]: I would also change error operations to something like: "xattr.Remove", "xattr.Get", ...

**Comment by kuba--** [comment]
@fd0 - do not forget about tests

**Comment by fd0** [comment]
Oh, indeed. I'll fix this.

**Comment by fd0** [comment]
Done, please review.

**Pull request merged**